backend/app/routers/product.py

```
@@ -2,13 +2,10 @@
 import app.schemas as schemas
 import app.models as models
 from app.database import get_db  # Database session dependency
-from app.models import User, Product, Brand, Order, OrderItem  # ORM models
+from app.models import User, Brand
```

Ruff (0.8.2)

`app.models.Brand` imported but unused

Remove unused import: `app.models.Brand`

(F401)